modules.json

```diff
@@ -1,90 +1,90 @@
 {
```

Comment:
Should this file be marked binary? It's just generated output right?

Reply:
It can be mergeable by humans.